week01/prove.md

```diff
@@ -13,7 +13,7 @@ Image of the drawing shapes generated by the assignment.
 
 ## Instructions
 
-1. Download `assignment.py` and `cse251turtle.py` from the `week01/assignment` folder in Github.  Create a folder on your computer that will contain these files. Read the header in the `assignment.py` file.  Refer to the Canvas document [directory structure](../overview/directory_structure.md).  Here is [GutHub Assignment's Folder](assignment)
+1. Download `assignment.py` and `cse251turtle.py` from the `week01/assignment` folder in Github.  Create a folder on your computer that will contain these files. Read the header in the `assignment.py` file.  Refer to the Canvas document [directory structure](../overview/directory_structure.md).  Here is [GitHub Assignments Folder](assignment)
```

Comment:
I think "Assignment's" should remain possessive in this case because the directory in question belongs to one assignment and only one assignment.

Reply:
Honestly, this one is a little rough, because the s should probably be left off all together. And now that I read it again, I missed a 'the that should be before the link.